packages/pluggableWidgets/rich-text-web/src/RichText.editorConfig.ts

```diff
@@ -0,0 +1,65 @@
+import {
+    Problem,
+    Properties,
+    hidePropertiesIn,
+    transformGroupsIntoTabs,
+    hidePropertyIn
+} from "@mendix/piw-utils-internal";
+import { RichTextPreviewProps } from "../typings/RichTextProps";
+
+const advancedModeItems: Array<keyof RichTextPreviewProps> = [
+    "enterMode",
+    "shiftEnterMode",
+    "spellChecker",
+    "codeHighlight",
+    "advancedContentFilter",
+    "allowedContent",
+    "disallowedContent"
+];
+const toolbarGroups: Array<keyof RichTextPreviewProps> = [
+    "toolsGroup",
+    "separatorGroup",
+    "stylesGroup",
+    "basicStylesGroup",
+    "clipboardGroup",
+    "colorsGroup",
+    "documentGroup",
+    "editingGroup",
+    "formsGroup",
+    "linksGroup",
+    "paragraphGroup",
+    "othersGroup",
+    "separator2Group"
+];
+
+export function getProperties(
+    values: RichTextPreviewProps,
+    defaultProperties: Properties,
+    platform: "web" | "desktop"
+): Properties {
+    if (values.preset !== "custom") {
+        hidePropertiesIn(defaultProperties, values, toolbarGroups.concat(["toolbarConfig", "advancedConfig"]));
+    }
+    if (values.toolbarConfig === "basic") {
+        hidePropertiesIn(defaultProperties, values, ["advancedConfig"]);
+    }
+    if (values.toolbarConfig === "advanced") {
+        hidePropertiesIn(defaultProperties, values, toolbarGroups);
+    }
+    if (values.advancedContentFilter === "auto") {
+        hidePropertiesIn(defaultProperties, values, ["allowedContent", "disallowedContent"]);
+    }
+    if (platform === "web") {
+        transformGroupsIntoTabs(defaultProperties);
+        hidePropertyIn(defaultProperties, values, "advancedMode");
```

Comment:
I think this check is incorrect. We should only show advanced mode in platform web and not the opposite.

Reply:
I think you misunderstood. You should hide this property when is not web, so in this case this line should be moved to the else. Because we dont want the toggle for advanced mode in Studio Pro.